docs/docs/docs/developer-resources/testing.md

```
@@ -186,6 +187,57 @@
+  
+5. Then exit
+
+  ```bash
+  exit
+  ```
+
+### Using VS Code Dev Container
+
+This applies to users running Talawa API in dev containers and VSCode.
+
+1. **WARNING:** This command will **DELETE** all data from each table in your database, administrator roles will be restored. Use with extreme caution.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)